packages/map-template/src/components/Map/Map.jsx

```diff
@@ -98,19 +102,49 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
         }
     }, [filteredLocations, filteredLocationsByExternalIDs, mapsIndoorsInstance]);
 
+    /*
+     * React to changes in bearing and pitch props and set them on the map if mapsIndoorsInstance exists.
+     */
+    useEffect(() => {
+        if (mapsIndoorsInstance) {
+            if (mapType === mapTypes.Google) {
+                if (bearing) {
+                    mapsIndoorsInstance.getMap().setHeading(parseInt(bearing));
+                }
+                if (pitch) {
+                    mapsIndoorsInstance.getMap().setTilt(parseInt(pitch));
+                }
+            } else if (mapType === mapTypes.Mapbox) {
+                if (bearing) {
+                    mapsIndoorsInstance.getMap().setBearing(parseInt(bearing));
+                }
+                if (pitch) {
+                    mapsIndoorsInstance.getMap().setPitch(parseInt(pitch));
+                }
+            }
+        }
+    }, [bearing, pitch, mapsIndoorsInstance]);
+
```

Comment:
In this `useEffect` we should do the same as in the `setVenue` function. So we do not need to check for Mapbox or Google maps, and we should use `getMapView()` instead.

Reply:
The indentation is wrong here on the if statements, please make sure the code is formatted correctly. 😄